pyiron/atomistics/job/atomistic.py

```diff
@@ -508,14 +508,18 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
         if not (self.structure is not None):
             raise AssertionError()
         snapshot = self.structure.copy()
-        snapshot.cell = self.get("output/generic/cells")[iteration_step]
-        snapshot.positions = self.get("output/generic/positions")[iteration_step]
-        indices = self.get("output/generic/indices")
-        if indices is not None:
+        snapshot.cell = self.output.cells[iteration_step]
+        snapshot.positions = self.output.positions[iteration_step]
+        indices = self.output.indices
+        if indices is not None and len(indices)>max([iteration_step,0]):
```

Comment:
Spacing for pep8 

`if indices is not None and len(indices) > max([iteration_step,0]):`